server.js

```diff
@@ -3054,6 +3054,14 @@ cache(function(data, match, sendBadge, request) {
       badgeData.text[1] = 'inaccessible';
       sendBadge(format, badgeData);
       return;
+    } else if (res.statusCode === 500) {
+      /* note:
+      david returns a 500 response for not found
+      e.g: https://david-dm.org/foo/barbaz/info.json
```

Comment:
Hm, that's unfortunate… it makes it impossible to distinguish between a bad project name and something being wrong with their servers.

Reply:
Yea, that's a good point. I'd be down with that. You could keep comments in the code about the behavior. Would also be a good idea to open an issue in the david-www repo to improve the API's erroring.